src/shadow-embed.js

```diff
@@ -548,6 +548,31 @@ export class ShadowDomWriterStreamer {
     return this.success_;
   }
 
+  /** @override */
+  get closed() {
+    return this.eof_;
+  }
+
+  /** @override */
+  get desiredSize() {
+    return 1; // noop
```

Comment:
I checked the tests and they are writing normally, definitely not writing one character at a time : P We are not using the `desiredSize` getter, so that makes sense. I expect that it doesn't have much meaning for us since our `WritableStreamDefaultWriter` instances don't have a backing stream, it's really just the `document` object, which doesn't expose the `WritableStream` methods that the `WritableStreamDefaultWriter` is supposed to proxy, like `desiredSize`.

That being said I looked for usages in the wild and found [this reference implementation](https://github.com/whatwg/streams/blob/7b26fa32263b7ceeebdb78783c316a3ec2d8e3ab/reference-implementation/to-upstream-wpts/transform-streams/general.js) which seems to expect the `desiredSize` value to be 0 when not writable to allow for backpressure, and 1 when writable, so it looks like we have a sane value.